TestCases/coolprop/pr.cfg

```diff
@@ -0,0 +1,247 @@
+% Author: Alberto Guardone                                                     %
+% Institution: Politecnico di Milano                                           %
+% Date: 2019.05.03                                                             %
+% File Version 6.2.0 "Falcon"                                                  %
+%                                                                              %
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+% ------------- DIRECT, ADJOINT, AND LINEARIZED PROBLEM DEFINITION ------------%
+%
+% Physical governing equations (EULER, NAVIER_STOKES,
+%                               FEM_EULER, FEM_NAVIER_STOKES, FEM_RANS, FEM_LES,
+%                               WAVE_EQUATION, HEAT_EQUATION, FEM_ELASTICITY,
+%                               POISSON_EQUATION)
+SOLVER= RANS
+%
+% Specify turbulence model (NONE, SA, SA_NEG, SST, SA_E, SA_COMP, SA_E_COMP)
+KIND_TURB_MODEL= SST
+%
+% Mathematical problem (DIRECT, CONTINUOUS_ADJOINT, DISCRETE_ADJOINT)
+MATH_PROBLEM= DIRECT
+%
+% Restart solution (NO, YES)
+RESTART_SOL= NO
+%
+% System of measurements (SI, US)
+% International system of units (SI): ( meters, kilograms, Kelvins,
+%                                       Newtons = kg m/s^2, Pascals = N/m^2,
+%                                       Density = kg/m^3, Speed = m/s,
+%                                       Equiv. Area = m^2 )
+% United States customary units (US): ( inches, slug, Rankines, lbf = slug ft/s^2,
+%                                       psf = lbf/ft^2, Density = slug/ft^3,
+%                                       Speed = ft/s, Equiv. Area = ft^2 )
+SYSTEM_MEASUREMENTS= SI
+%
+% -------------------- COMPRESSIBLE FREE-STREAM DEFINITION --------------------%
+%
+% Mach number (non-dimensional, based on the free-stream values)
+MACH_NUMBER= 1E-9
+%
+% Angle of attack (degrees, only for compressible flows)
+AOA= 0.0
+%
+% Side-slip angle (degrees, only for compressible flows)
+SIDESLIP_ANGLE= 0.0
+%
+% Init option to choose between Reynolds (default) or thermodynamics quantities
+% for initializing the solution (REYNOLDS, TD_CONDITIONS)
+INIT_OPTION= TD_CONDITIONS
+%
+% Free-stream option to choose between density and temperature (default) for
+% initializing the solution (TEMPERATURE_FS, DENSITY_FS)
+FREESTREAM_OPTION= TEMPERATURE_FS
+%
+% Free-stream pressure (101325.0 N/m^2, 2116.216 psf by default)
+FREESTREAM_PRESSURE= 904388
+%
+% Free-stream temperature (288.15 K, 518.67 R by default)
+FREESTREAM_TEMPERATURE= 542.13
+%
+% Compressible flow non-dimensionalization (DIMENSIONAL, FREESTREAM_PRESS_EQ_ONE,
+%                              FREESTREAM_VEL_EQ_MACH, FREESTREAM_VEL_EQ_ONE)
+REF_DIMENSIONALIZATION= DIMENSIONAL
+
+% ---- IDEAL GAS, POLYTROPIC, VAN DER WAALS AND PENG ROBINSON CONSTANTS -------%
+%
+% Fluid model (STANDARD_AIR, IDEAL_GAS, VW_GAS, PR_GAS,
+%              CONSTANT_DENSITY, INC_IDEAL_GAS, INC_IDEAL_GAS_POLY)
+FLUID_MODEL= PR_GAS
```

Comment:
This doesn't use the coolprop library

Reply:
You are right, I removed this file. 
Thank you.